tools/scripts/ansible/setup-playbook.yml

```diff
@@ -196,3 +196,5 @@
         --keep --redirect --uir --hsts --staple-ocsp --must-staple
       args:
         creates: /etc/letsencrypt/renewal/{{ inventory_hostname }}.conf
+
+- import_playbook: backup-db-playbook.yml
```

Comment:
Hm, I don't think this import is needed here. You can just run the backup-db-playbook by itself.